cleaners/safari.xml

```diff
@@ -19,36 +19,64 @@
     along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 -->
-<cleaner id="safari" os="windows">
+<cleaner id="safari">
   <label>Safari</label>
   <description>Web browser</description>
   <option id="cache">
     <label>Cache</label>
     <description>Delete the web cache, which reduces time to display revisited pages</description>
-    <action command="delete" search="file" path="$localappdata\Apple Computer\Safari\Cache.db"/>
+    <group os="windows">
+      <action command="delete" search="file" path="$localappdata\Apple Computer\Safari\Cache.db"/>
+    </group>
+    <group os="darwin">
+      <action command="delete" search="file" path="~/Library/Safari/Cache.db"/>
+    </group>
   </option>
   <option id="cookies">
     <label>Cookies</label>
     <description>Delete cookies, which contain information such as web site preferences, authentication, and tracking identification</description>
-    <action command="delete" search="file" path="$APPDATA\Apple Computer\Safari\Cookies\Cookies.plist"/>
-    <action command="delete" search="walk.files" path="$localappdata\Apple Computer\Safari\LocalStorage\"/>
+    <group os="windows">
+      <action command="delete" search="file" path="$APPDATA\Apple Computer\Safari\Cookies\Cookies.plist"/>
+      <action command="delete" search="walk.files" path="$localappdata\Apple Computer\Safari\LocalStorage\"/>
+    </group>
+    <group os="darwin">
+      <action command="delete" search="file" path="~/Library/Safari/Cookies.plist"/>
+      <action command="delete" search="walk.files" path="~/Library/Safari/LocalStorage/"/>
+    </group>
   </option>
   <option id="history">
     <label>History</label>
     <description>List of visited web pages</description>
-    <action command="delete" search="file" path="$APPDATA\Apple Computer\Safari\History.plist"/>
-    <action command="delete" search="file" path="$APPDATA\Apple Computer\Safari\LastSession.plist"/>
-    <action command="delete" search="walk.files" path="$localappdata\Apple Computer\Safari\History\"/>
-    <action command="delete" search="walk.files" path="$localappdata\Apple Computer\Safari\Webpage Previews\"/>
+    <group os="windows">
```

Comment:
`<group>` is not supported in the XSD or the Cleaner.py code. What are you trying to do? When running on one operating system, the files from the other OS will simply not exist and be ignored. If you just want to organize, please use XML comments instead.

Also in the future I plan to improve this by defining cleaner-level variables that would look something like this
````
<var os="darwin" name="safari" value="~/Library/Safari/" />
<var os="windows" name="safari" value="%APPDATA%\Apple Computer\Safari\" />
````
Then each action, like cleaning History.plist, can reference this variable.